apis/theme/src/colors/color-scale.js

```diff
@@ -0,0 +1,84 @@
+import Color from './color';
+
+const NaNColor = new Color(0xd2d2d2);
```

Comment:
Shouldn't the NaN color come from the applied theme?

Reply:
This match how it is done in the sense client.
I don't think it is important as it should not be used if the scale is correctly defined.
(If it should be from the theme what color in the theme should it use?)

Reply:
Would be nice if we didn't just copy the mistakes from previous code ;) - NaN/Null color is specified in the [theme](https://github.com/qlik-oss/nebula.js/blob/master/apis/theme/src/themes/base.json#L9)

Reply:
done.
Is it relevant to have a warning when the nan color is used to indicate that there is a problem with the theme?

Reply:
No need for a warning specific for the NaN color right now, but if we see a lot of issues of poorly formatted themes then having a theme validator would be nice to have in the future.